sheu@chromium.org: trimmed from 3.4 kernel patch

From a282f22565b9e0d1aefdbe1b2956d11c21b81810 Mon Sep 17 00:00:00 2001
From: Subash Patel <subash.ramaswamy@linaro.org>
Date: Tue, 22 May 2012 20:18:30 +0100
Subject: [PATCH] Media: Exynos: Header file support for G-Scaler driver

This commit adds the header files required to define the controls
of the new G-Scaler driver.

Change-Id: Iea9d01e18870501758326750c3b0051df9ec139f
Signed-off-by: Subash Patel <subash.ramaswamy@linaro.org>
Signed-off-by: Kiran AVND <kiran.avnd@samsung.com>
---
 arch/arm/mach-exynos/include/mach/videonode.h |   32 +++++++++++++++++++++++++
 include/linux/videodev2.h                     |    2 +
 2 files changed, 34 insertions(+), 0 deletions(-)
 create mode 100644 arch/arm/mach-exynos/include/mach/videonode.h

diff --git a/include/linux/videodev2.h b/include/linux/videodev2.h
index 9bbd3b3..17715dd 100644
--- a/include/linux/videodev2.h
+++ b/include/linux/videodev2.h
@@ -361,9 +361,11 @@ struct v4l2_pix_format {
 #define V4L2_PIX_FMT_NV12M   v4l2_fourcc('N', 'M', '1', '2') /* 12  Y/CbCr 4:2:0  */
 #define V4L2_PIX_FMT_NV21M   v4l2_fourcc('N', 'M', '2', '1') /* 21  Y/CrCb 4:2:0  */
 #define V4L2_PIX_FMT_NV12MT  v4l2_fourcc('T', 'M', '1', '2') /* 12  Y/CbCr 4:2:0 64x32 macroblocks */
+#define V4L2_PIX_FMT_NV12MT_16X16 v4l2_fourcc('V', 'M', '1', '2') /* 12  Y/CbCr 4:2:0 16x16 macroblocks */
 
 /* three non contiguous planes - Y, Cb, Cr */
 #define V4L2_PIX_FMT_YUV420M v4l2_fourcc('Y', 'M', '1', '2') /* 12  YUV420 planar */
+#define V4L2_PIX_FMT_YVU420M v4l2_fourcc('Y', 'V', 'U', 'M') /* 12  YVU420 planar */
 
 /* Bayer formats - see http://www.siliconimaging.com/RGB%20Bayer.htm */
 #define V4L2_PIX_FMT_SBGGR8  v4l2_fourcc('B', 'A', '8', '1') /*  8  BGBG.. GRGR.. */
-- 
1.7.8.6

